projectDocs/translating/crowdin.md

```diff
@@ -83,6 +80,9 @@ Ensure that it has at least the translations scope.
 Then paste this into the user prompt.
 This will be saved in ~/.nvda_crowdin for future use.
 
+If your language team has more than one translator who may be downloading, translating and uploading at the same time, it is important that once you have downloaded the file, that you save a copy before you start translating with Poedit.
+ This then allows you to provide l10nUtil with this original file when uploading, so that it can just upload only what has changed, which will avoid accidentally overriding another translator's work.
```

Comment:
```suggestion
This then allows you to provide l10nUtil with this original file when uploading, so that it can just upload only what has changed, which will avoid accidentally overriding another translator's work.
```